getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -20,13 +20,12 @@ Prerequisites on the target device
 **********************************
 
 The target device can be any of the SOF-supported platforms,
-e.g. MinnowBoard, Up^2, Asus T100, Chromebooks)
+e.g. MinnowBoard, Up^2, Asus T100, Chromebooks.
```

Comment:
Can you please fix line 15 in the introduction? It says "though only one branch can be checked out at a time" whereas the FIRST line of `git help worktree` says "...  allowing you to check out more than one branch at a time"